package.json

```diff
@@ -9,8 +9,8 @@
     "clean": "find packages -name dist -or -name lib -or -name node_modules -type d | xargs rm -rf",
     "commit": "git-cz",
     "commitmsg": "validate-commit-msg",
-    "deploy:site": "npm run build && npm run test && npm run deploy:site:dist",
-    "deploy:site:dist": "gh-pages -d packages/site/dist -m 'chore(site): Deploy' -r https://github.com/mineral-ui/mineral-ui.github.io.git -b master",
+    "deploy:site": "npm run build && npm run test && npm run netlify",
+    "netlify": "netlify deploy",
```

Comment:
we may need two scripts for a while to deploy both sites in parallel until we get DNS migrated successfully